lift.css

```diff
@@ -0,0 +1,362 @@
+:root{
+    --primary-color: #cfeefb;
+    --grey-color: gray;
+    --green-success:  rgb(27, 170, 27);
+    --red-failed:rgb(253, 37, 37);
+    --yellow-warning:#ff9800;
+
+}
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    margin: 0px
+}
+
+.index-card{
+    width: 100%;
+    height: 100%;
+    margin:10px;
+    padding: 2rem;
+    text-align: center;
+}
+
+.code{
+    text-align: center;
+}
+
+aside{
+   width: 20%;
+    margin-right: 25px;
+   
+}
+
+.wrapper{
+    display: flex;
+}
+
+.wrapper-nav{
+    display: grid;
+    grid-template-columns: 1fr 4fr;
+}
+
+.wrapper-input{
+    border: 0.5px solid black;
+    margin: 0.5rem;
+    padding: 1rem;
+}
+
+.center{
+    text-align: center;
+}
+
+.gray{
+    color: var(--grey-color);
+}
+
+input{
+    margin: 0.5rem;
+}
+
+.input-label{
+    display: flex;
+    flex-direction: column;
+    font-size: medium;
+    font-weight: bold;
+    margin: 0.05rem;
+}
+
+.list-non-bullet{
+    list-style: none;
+    padding-inline-start: 0px ;
+}
+
+.list-item-inline{
+    display: inline; 
+    padding: 0rem 0.5 rem;
+}
+
+.list-stacked{
+    padding: 0.25rem;
+    border: 0.5px solid black;
+}
+
+.navigation{
+    background-color: var(--primary-color);
+    padding: 0.25rem;
+    display: flex;
+    align-items: center;
+    width: 100%;
+}
+
+.navigation .nav-brand{
+    font-weight: bold;
+    font-size: 1.1rem;
+    display: inline;
+}
+
+.navigation .nav-pills{
+    margin-left: auto;
+
+}
+ .link{
+     text-decoration: none;
+     padding: 0.5rem 1rem;
+     margin-top: 1rem;
+     margin-bottom: 1rem;
+ }
+ .link-primary{
+     background-color: var(--red-failed);
+     border-radius: 0.5rem;
+     color: white
+
+ }
+
+ .link-secondary{
+     color: var(--red-failed);
+     border-radius: 0.5rem;
+     border: 1px solid var(--red-failed)
+ }
+
+ .link-active{
+     font-weight: bold;
+     
+ }
+
+ .row{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.avatar{
+    /* vertical-align: middle; */
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+    margin-right: 10px;
+    margin-left: 5px;
+}
+
+.size-s{
+    width: 2.5rem;
+    height: 2.5rem;     
+}
+
+.size-m{
+    width: 5rem;
+    height: 5rem;  
+}
+
+.size-l{
+    width:7.5rem;
+    height: 7.5rem;
+}
+
+.badge{
+    border: 0.5px solid red;
+    background-color: red;
+    padding: 0.3rem;
+}
+
+.badge-thumbnail{
+    position: relative;
+}
+
+.badge-icon{
+    position: absolute;
+    top:0rem;
+    right:0rem;
+}
+
+.card{
+    display: flex;
+    padding: 1rem;
+    border: 1px solid black;
+    flex-direction: column;
+   max-width: 250px;    
+}
+
+.card-shadow{
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
```

Comment:
The Shadow is very light and not clearly visible. 
You can use this instead **box-shadow: 0.5rem 0.5rem 1rem rgb(0 0 0 / 45%)** or adjust it accordingly.

Reply:
Thanks Amarjit, I have fixed the card-shadow.